src/microbit/__model/image.py

```diff
@@ -75,7 +112,6 @@ class Image:
     # https://microbit-micropython.readthedocs.io/en/latest/image.html

```

Comment:
This link is set to show the docs on `latest` which might be updated. Do you think you can fix the version, so if `latest` gets updated our comments are correct?